test/browser/package.json

```diff
@@ -4,12 +4,13 @@
   "module": "true",
   "scripts": {
     "test": "pnpm run test:webdriverio && pnpm run test:playwright",
-    "test:webdriverio": "PROVIDER=webdriverio node --test specs/",
-    "test:playwright": "PROVIDER=playwright node --test specs/",
+    "test:webdriverio": "cross-env PROVIDER=webdriverio node --test specs/",
+    "test:playwright": "cross-env PROVIDER=playwright node --test specs/",
     "coverage": "vitest --coverage.enabled --coverage.provider=istanbul"
   },
   "devDependencies": {
     "@vitest/browser": "workspace:*",
+    "cross-env": "^7.0.3",
```

Comment:
We don't need `cross-env`, this functionality is built into pnpm

Reply:
The thing is that the ci failed before in the previous commits! Any thoughts?

```

> @vitest/monorepo@0.29.8 test:browser:webdriverio D:\a\vitest\vitest
> npm -C test/browser run test:webdriverio


> test:webdriverio
> PROVIDER=webdriverio node --test specs/

'PROVIDER' is not recognized as an internal or external command,
operable program or batch file.
 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.

```

Reply:
let me try pnpm instead of npm